website/data/pages/typography.mdx

```diff
@@ -1,3 +1,17 @@
 ---
 title: Typography
----
\ No newline at end of file
+description: 
+---
+
+# Typography
+
+Typography is an important part of the Design System that brings consistency across experiences and platforms. Good typography rules help present content clearly and efficiently.
+
+## Typeface
+
+Strapi Design System uses system fonts to provide the best experience on every Operating System (OS). Due to that chance, a performance issue is also reduced. <br/> <br/>
+Moreover, each level of the Type Scale is defined by the Greek alphabet to abstract all context from each level and developed in rem instead of px. Thus, any usecase should be covered by the scale.
+
+![Typeface](/typeface.png)
```

Comment:
The typographies are not the valid ones

Reply:
@maevalienard could you pls inform me where to find the valid typography? Thanks.

Reply:
Replaced it by the actual typographies from the [UI Kit](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64017)